layouts/default.vue

```diff
@@ -74,11 +84,38 @@ export default {
                     href:
                         'https://fonts.googleapis.com/css2?family=Noto+Serif+TC:wght@400;500;600;700&family=Source+Sans+Pro:wght@400;600;700&display=swap',
                 },
-
                 {
                     rel: 'canonical',
                     href: 'https://tw.pycon.org' + this.$route.path,
                 },
+                {
+                    rel: 'alternate',
+                    hreflang: 'x-default',
+                    href: 'https://tw.pycon.org/2021/en-us',
+                },
+                {
+                    rel: 'alternate',
+                    hreflang: 'en-us',
+                    href: 'https://tw.pycon.org/2021/en-us',
+                },
+                {
+                    rel: 'alternate',
+                    hreflang: 'zh-hant-tw',
+                    href: 'https://tw.pycon.org/2021/zh-hant',
+                },
+            ],
+            script: [
+                {
+                    type: 'application/ld+json',
+                    json: {
+                        '@context': 'https://schema.org',
+                        '@type': 'Organization',
+                        name: 'PyCon Taiwan 2021',
+                        url: 'https://tw.pycon.org/2021/en-us',
```

Comment:
Please also add [`sameAs`](https://schema.org/sameAs).
```suggestion
                        url: 'https://tw.pycon.org/2021/en-us',
                        sameAs: [ 
                            "https://www.facebook.com/pycontw/",
                            "https://twitter.com/PyConTW/",
                            "https://www.linkedin.com/company/pycontw",
                            "https://www.youtube.com/PyConTaiwan",
                            "https://instagram.com/pycontw/",
                            "https://github.com/pycontw/",
                            "https://pycontw.blogspot.com/",
                        ],
}
```